doc/source/whatsnew/v1.1.0.rst

```diff
@@ -220,6 +220,8 @@ Other enhancements
   This can be used to set a custom compression level, e.g.,
   ``df.to_csv(path, compression={'method': 'gzip', 'compresslevel': 1}``
   (:issue:`33196`)
+- :meth:`~pandas.core.groupby.GroupBy.transform` and :meth:`~pandas.core.groupby.GroupBy.aggregate` has gained ``engine`` and ``engine_kwargs`` arguments that supports executing functions with ``Numba`` (:issue:`32854`, :issue:`33388`)
```

Comment:
why is this added? from resolving a merge conflict maybe?

Reply:
Yeah that's why I put it there, but it seems to be unnecessary.